core/tests/webdriverio_desktop/extensions.js

```diff
@@ -224,13 +223,13 @@ describe('Interactions', function() {
     await explorationEditorMainTab.setContent(await forms.toRichText(
       'Draw a complete graph with the given vertices.'), true);
     var graphDictForInput = {
-      vertices: [[277, 77], [248, 179], [405, 144]]
+      vertices: [[-10, -50], [-39, 72], [118, 17]]
```

Comment:
Can you please write an explanation here which you explained me in the meeting in order to keep it as a future reference?

Reply:
In protractor the 0,0 coordinate lie at the top left corner of selected element while in webdriverio it lies at the centre of the selected element so we need to change the coordinates accordingly.